aws/resource_aws_db_instance_test.go

```diff
@@ -3337,6 +3370,28 @@ resource "aws_db_instance" "bar" {
 `)
 }
 
+func testAccAWSDBInstanceConfig_MajorVersionOnly(engine, engineVersion string) string {
+	return composeConfig(testAccAWSDBInstanceConfig_orderableClassMysql(), fmt.Sprintf(`
+resource "aws_db_instance" "test" {
+  allocated_storage       = 10
+  backup_retention_period = 0
+  engine                  = %[1]q
```

Comment:
This (and the equivalent in `aws_rds_cluster`) is causing **tflint** to complain: https://github.com/terraform-linters/tflint-ruleset-aws/blob/v0.4.0/docs/rules/aws_db_instance_invalid_engine.md.
Maybe try using an intermediate HCL local?

```hcl
locals {
  engine = %[1]q
}

resource "aws_db_instance" "test" {
  engine = local.engine
}
```

Reply:
Good call. Thank you.